code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -365,25 +365,27 @@
 	H << "<span class='warning'><b>You crumple in agony as your flesh wildly morphs into new forms!</b></span>"
 	H.visible_message("<b>[H]</b> falls to the ground and screams as their skin bubbles and froths!") //'froths' sounds painful when used with SKIN.
 	H.Weaken(3)
-	sleep(30)
-	//var/list/blacklisted_species = list(
-
-	var/list/possible_morphs = list()
-	for(var/type in subtypesof(/datum/species))
-		var/datum/species/S = type
-		if(initial(S.blacklisted))
-			continue
-		possible_morphs += S
-	var/datum/species/mutation = pick(possible_morphs)
-	if(prob(90) && mutation && H.dna.species != /datum/species/golem && H.dna.species != /datum/species/golem/adamantine)
-		H << "<span class='danger'>The pain subsides. You feel... different.</span>"
-		H.set_species(mutation)
-		if(mutation.id == "slime")
-			H.faction |= "slime"
+	spawn(30)
```

Comment:
lower overhead; however it mostly relates to CPU thrashing from a lot of spawns at the same time. even if spammed this would be spread out so I suppose its fine